spec/unit/rack/context_filter_spec.rb

```diff
@@ -38,7 +38,7 @@ def env_for(url, opts = {})
       { 'HTTP_USER_AGENT' => 'Bingo Agent' }
     end
 
-    it "adds User-Agent to the context" do
+    it "adds UserAgent to the context" do
```

Comment:
My bad, assumed the test referred to the `userAgent` data field in the API. https://airbrake.io/docs/api/#create-notice-v3